client/__tests__/util/typedCrossfilter/crossfilter.test.js

```diff
@@ -11,7 +11,8 @@ const someData = [
     tip: 100,
     type: "tab",
     productIDs: ["001"],
-    coords: [0, 0]
+    coords: [0, 0],
+    nonFiniteTest: 0.0
```

Comment:
[Nit] nonFiniteTest -> nonFinite - none of the other keys have the Test suffix

Reply:
fixed